src/os/FileStore.cc

```diff
@@ -1891,8 +1915,16 @@ void FileStore::_do_op(OpSequencer *osr, ThreadPool::TPHandle &handle)
     dout(5) << "_do_op done stalling" << dendl;
   }
 
-  osr->apply_lock.Lock();
-  Op *o = osr->peek_queue();
+  do {
+    osr->apply_lock.Lock();
+    o = osr->peek_queue();
+    if (o->flags & OP_SKIP_WBTHROTTLE == 0) {
+      osr->apply_lock.Unlock();
+      wbthrottle.throttle();
```

Comment:
In fact, only FileStore::_write count into wbthrottle. For xattr & omap don't count this.
For xattr/omap write-operations, it don't count into wbthrottle.